### PR TITLE
[SYCLomatic] Migrate cublas[S|D|C|Z]gemm_v2[_64] with rewriter

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/runtime/dpct-rt/include/util.hpp [hunk 1]
```diff
@@ -1025,6 +1025,17 @@ class args_selector<n_nondefault_params, n_default_params, R(Ts...)> {
 #define DPCT_EXPORT
 #endif
 
+/// Cast a "rvalue reference to a temporary object" to an "lvalue reference to
+/// that temporary object".
+/// The returned lvalue reference is available only before the last step in
+/// evaluating the full-expression that contains this function call.
+/// \param [in] temporary_object The rvalue reference to a temporary object.
+/// \returns The lvalue reference to that temporary object.
+template <typename T>
```
Comment (zhimingwang36) [inline clang/runtime/dpct-rt/include/util.hpp]: suggest move this help to lib_common_utils.hpp for library migration only. only available in USM-none.

Comment (zhiweij1) [reply clang/runtime/dpct-rt/include/util.hpp]: done

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**